articles/dd-you-did-what.md

```diff
@@ -0,0 +1,65 @@
+
+Organizational tools are remote and colocated pair programming and knowledge sharing sessions. They help to build the team's knowledge base stronger and boost up their confidence and mutual trust to perform as one team.
+
+### 3. What are the main benefits?
+**Scalability** of the team is close to limitless since it is not dependent on one location. **Finding skilled team members** as the best expert for the job **is easier** in DD setup since it creates an access to larger employees pool. On another hand, it helps to **keep in close proximity to the Market and the Customer**. **Cost benefits** are positively affected since it allows a project to hire team members from other locations with **lower rates** but keeping the **same quality** of and desired expertise level.
+DD with its distributed locations **increases competitiveness on global market** and allowing companies to fulfill growing market interest. Companies have an opportunity to **meet new people**, travel and **exercise knowledge sharing** through the whole organization.
+It also decouples employees and their work from their physical location which means **no location boundaries**. They can do home office or any office (across the whole organization).
+Since DD setup is vendor's internal organization process there is **no overhead for Customer**.
+
+### 4. What are the challenges and how to overcome them?
+##### Language barrier - bilingual
+English, as a common language of communication, is one of the best practices for Distributed development setup. But since it covers global market, most common challenge is the language barrier. This is mostly coming from Customer side which is understandable and realistic. Simply, there are no exclusive conditions that all of the business representatives should know to speak English. This can be solved by using bilingual approach for all shared resources, better plan and facilitate meetings where there are participants that use different language for communication.
+
+##### Different time zones - collocated kickoffs and time management
+DD setup can also work with different time zones. Collocated project kick-off helps to get the all team members to meet each other better and to generate one team feel. Also at the kick off time planning and management is important, so the team can use the time overlaps in the best possible way as the project progresses.
+
+##### On-sight support - proxy
+Projects usually provide production on-sight support for Customers. This could have limitations if the team is distributed on locations further from the Customer or in other countries, and also from a legal perspective. This is solved by introducing proxy roles on the side of the team which is closest to the Customer. This role allows the whole team to be involved and contribute to the production support on their project.
+
+##### Cultural differences - collocated kick off and regular visits
```

Comment:
collocated kick off is certainly important - but I think starting from here it is mentioned too often (it already the third time and 2 more will follow). 

consider reducing the number you mention "collocated kick off" to avoid the impression of repeating yourself

Reply:
I remodeled some parts of the text so that I don't repeat myself :) I also merged Cultural differences and Initial mindset since it has a similar solution.